product_form_sale_link/__manifest__.py

```diff
@@ -1,14 +1,15 @@
+# Copyright 2021 Xtendoo
```

Comment:
Don't add yourself to authors for a migration.

You should add just a line in CONTRIBUTORS